cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -142,6 +143,8 @@ func generateInitramfsMounts() (err error) {
 		err = generateMountsModeInstall(mst)
 	case "run":
 		err = generateMountsModeRun(mst)
+	case "cloudimg-rootfs":
+		err = generateMountsModeClassic(mst, "cloudimg-rootfs")
```

Comment:
do we expect to have different labels eventually? if not, I would rather this not be a parameter but rather just a constant defined somewhere here with a comment that "this is the expected value" etc